AdminUi/src/AdminUi/ClientApp/src/app/components/quotas/assign-quotas-dialog/assign-quotas-dialog.component.ts

```diff
@@ -73,7 +109,7 @@ export class AssignQuotasDialogComponent {
             period: this.period!
         };
 
-        this.dialogRef.close(quota);
+        this.createIdentityQuota(quota);
```

Comment:
Now the assign-quotas-dialog always creates quota for an identity. But what if we want to use it to create quota for a tier?